index.html

```diff
@@ -0,0 +1,265 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://www.dafontfree.net/embed/Y29jb2dvb3NlLXByby10aGluLXJlZ3VsYXImZGF0YS84NS9jLzE3NjcxMS9Db2NvZ29vc2UgUHJvIFRoaW4tdHJpYWwudHRm"
+    rel="stylesheet" type="text/css" />
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+  <section class="head">
+
+    <header class="header">
+
+      <nav>
+
+        <div class="topNav">
+
+          <ul class="navSocials">
+            <li><a href="#"><img src="https://img.icons8.com/color/48/000000/facebook-new.png" alt="facebook" /></a>
+            </li>
+            <li><a href="#"><img src="https://img.icons8.com/color/48/000000/twitter--v1.png" alt="twitter" /></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+
+          </ul>
+
+        </div>
+
+        <div class="lowerNav">
+
+          <div class="lowerNavContainer">
+
+            <div class="logo"><img src="images/logo.jpg" alt="logo">
+              <p>Nobilis</p>
+            </div>
+
+            <div class="openMenu"><img src="https://img.icons8.com/ios-filled/50/000000/menu--v1.png" alt="open menu" />
+            </div>
+
+            <ul class="mainMenu">
+              <li><a href="about.html">About Us</a></li>
+              <li><a href="#program">Program</a></li>
+              <li><a href="#lecturers">Lecturers</a></li>
+              <li class="closeMenu"><img src="https://img.icons8.com/ios-glyphs/30/000000/delete-sign.png"
+                  alt="close menu" /></li>
+            </ul>
+
+          </div>
+
+        </div>
+
+      </nav>
+
+      <div class="header-container">
+
+        <div class="bootcamp-headings">
+
+          <div class="title">
+            <h1 class="hello">"Hello Nobilis"</h1>
+            <h2 class="course-title">Advanced Web development <br>Summer bootcamp <br>2022 </h2>
+          </div>
+
+          <div class="summary">
+            <p>Launch a career as a web developer by learning <br> HTML5, CSS3, responsive design, Javascript and
+              more!
+            </p>
+          </div>
+
+          <div class="timeline">
+            <p>From 2022.06.08</p>
+          </div>
+
+
+        </div>
+
+      </div>
+
+    </header>
+
+  </section>
+
+  <main>
+
+    <section class="program">
+
+      <div class="main-program" id="program">
+
+        <div class="main-program-contents">
+
+          <div>
+            <h2 class="main-program-head">Main program</h2>
+          </div>
+
+          <div>
+            <hr class="line">
+          </div>
+
+          <div class="card">
```

Comment:
- [ ] Well done on implementing the other requested changes from the other reviewers but as they requested you to change to the background of the main program section and kindly fix the design of the cards for the mobile screen to stick as much as you can to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163588456-75ea0026-3b9f-4e78-97a3-5ebd0341c9b4.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163588629-5438f354-9c3c-484d-acdc-dd3881eaaaf5.png)